openid-sharedsignals-framework-1_0.md

```diff
@@ -1123,6 +1140,19 @@ description
   This is useful in multi-stream systems to identify the stream for human actors. The
   transmitter MAY truncate the string beyond an allowed max length.
 
+inactivity_ttl
+
+> **Transmitter-Supplied**, OPTIONAL. The refreshable time-to-live (TTL) of the stream in seconds, after which the Transmitter MAY either pause, disable, or delete the stream if it has not received any Receiver-initiated communication (defined below) in that duration. The syntax is the same as that of {{EXPIRES_IN}}.
+>
+> For streams created with the PUSH {{RFC8935}} delivery method, the Transmitter MUST refresh the TTL whenever:
```

Comment:
"refresh the TTL" could be ambiguous. Can we be more precise? we could alter the description in line 1145 to read: "the refreshable time-to-live (TTL) of the stream in seconds, since the last activity from the Receiver as described below, was conducted. After the TTL, the Transmitter MAY..."

This sentence can then read: "Receiver activity can be any of:"